src/collective/upload/profiles/default/cssregistry.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <object name="portal_css" meta_type="Stylesheets Registry">
-  <stylesheet title="" cacheable="True" compression="safe" cookable="True"
-    enabled="1" expression="" id="++resource++collective.upload/css/bootstrap.min.css" media="screen"
-    rel="stylesheet" rendering="import"/>
+  <stylesheet title="" cacheable="True" compression="none" cookable="True"
```

Comment:
@rodfersou installing this resource is a job of collective.js.bootstrap; check collective.cover for instance.